src/Resources/views/storefront/component/listing/filter/filter-property-select.html.twig

```diff
@@ -2,5 +2,23 @@
 
 {% block component_filter_property_select_preview %}
     {% set displayType = element.displayType %}
-    {{ parent() }}
+
+    <span class="filter-property-select-preview"
+          title="{{ name }}"
+          {% if displayType == 'media' and media %}
+          style="background-color: {{ color }};background-image: url('{{ media }}')"
+          {% elseif displayType == 'color' and color %}
+          style="background-color: {{ color }};background-image:none"
+          {% endif %}
```

Comment:
Does that mean displayType = 'none' is the Shopware default image?

Reply:
Exactly. `none` will be used in case there is no URL and no image configured.